drivers/spi/spi_ll_stm32.c

```diff
@@ -367,15 +367,34 @@ static int spi_stm32_shift_frames(SPI_TypeDef *spi, struct spi_stm32_data *data)
 	return spi_stm32_get_err(spi);
 }
 
-static void spi_stm32_complete(struct spi_stm32_data *data, SPI_TypeDef *spi,
-			       int status)
+static void spi_stm32_subghzspi_nss_control(const struct device *dev, bool on)
 {
+#if defined(CONFIG_SOC_SERIES_STM32WLX)
```

Comment:
I'd prefer using a dedicated compat and avoid series related switches (as much as possible)

Reply:
Cool, this is now a `#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_spi_subghz)`

Reply:
Another (untested) option might be to force usage the hardware NSS support (this require selecting SPI_STM32_USE_HW_SS and not defining a NSS pin in DT), that way the NSS pin is implicit and does not require any change to the SPI driver.

Reply:
Yeah I tried that, does not seem to work unfortunately. Double checked that `LL_SPI_SetNSSMode(spi, LL_SPI_NSS_HARD_OUTPUT);` gets called, it does, but then the device does not respond. Sorry.